src/SideBar.test.tsx

```diff
@@ -238,22 +212,4 @@ describe('SideBar', () => {
     screen.getByTestId('desktop-sidebar-close').click();
     expect(mockProps.onDesktopCloseClick).toHaveBeenCalledTimes(1);
   });
-
-  test('calls onClickSearchHistory when mobile search history button is clicked', () => {
-    vi.mocked(useIsMobileLayout).mockReturnValue(true);
-
-    render(<SideBar {...mockProps} isMobileSideBarOpen={true} />);
-
-    screen.getByTestId('mobile-search-history').click();
-    expect(mockProps.onClickSearchHistory).toHaveBeenCalledTimes(1);
-  });
-
-  test('calls onClickSearchHistory when desktop search history button is clicked', () => {
-    vi.mocked(useIsMobileLayout).mockReturnValue(false);
-
-    render(<SideBar {...mockProps} isDesktopSideBarOpen={true} />);
-
-    screen.getByTestId('desktop-search-history').click();
-    expect(mockProps.onClickSearchHistory).toHaveBeenCalledTimes(1);
-  });
```

Comment:
Once the search modal is finished and wired up, we can do a more legit, integration-style test that verifies that the expected modal is opened when the button is clicked.